manager/src/main/java/org/openremote/manager/security/KeyStoreServiceImpl.java

```diff
@@ -144,6 +176,17 @@ public TrustManagerFactory getTrustManagerFactory() throws Exception {
 
 		return tmf;
 	}
+	public KeyStore createKeyStore(Path path) throws Exception {
+		KeyStore keystore = ((KeyStore)null);
+		// Save the newly created KeyStore
+		try (OutputStream os = new FileOutputStream(path.toFile())) {
+			keyStore.store(os, getKeyStorePassword());
+			keystore.load(new FileInputStream(path.toFile()), getKeyStorePassword());
+		} catch (Exception saveException) {
+			getLogger().severe("Couldn't store KeyStore to Storage! " + saveException.getMessage());
+		}
+		return keystore;
+	}
```

Comment:
Don't think we should be creating key stores, user supplies them or doesn't. If this is just for tests then move it there. FYI: Groovy tests can access private/protected/package visible methods.

Reply:
The issue with this is that, if we do so, then we will have to somehow fail fast and hard during configuration of an agent to use mTLS. I'm not exactly sure what the difference would be if we do create the keystore, as we can fail silently saying that the alias requested does not exist in the keystore. Most users will ignore the functionality, the files will be basically empty, and when they want to use them, they're available. A lot of users also may not know how to exactly create a keystore, and using a GUI for it makes it a lot easier, and letting us do the heavy lifting with creating the keystores may be something convenient for them. Everything is documented as well, and I'll update the documentation when this PR is done.